config/chefstyle.yml

```diff
@@ -713,4 +713,9 @@ Chef/Ruby/GemspecRequireRubygems:
 Chef/Ruby/RequireNetHttps:
   Description: net/https is deprecated and just includes net/http and openssl. We should include those directly instead
   Enabled: true
-  VersionAdded: '1.3.0'
\ No newline at end of file
+  VersionAdded: '1.3.0'
+
+Chef/Ruby/LegacyPowershellOutMethods:
+  Description: Use powershell_exec!/powershell_exec instead of the slower legacy powershell_out!/powershell_out methods.
+  Enabled: true
+  VersionAdded: '1.6.0'
```

Comment:
why does your editor not properly end files? :(

Reply:
I turned it on now just for you. It causes a lot of issues with other projects that get grumpy about newlines